### ISSUE TITLE
LDAP Filter has no effect

### ISSUE THREAD

**Issue by obstschale** [freescout-helpdesk/freescout#1299]
We are using FreeScout since last friday in production and it just works. :+1: We are using the LDAP module for authentication. However, my SysAdmin told me that the LDAP filter in does not work.

![CleanShot 2021-06-29 at 23 04 05](https://user-images.githubusercontent.com/1058710/123867741-0675b600-d92f-11eb-9d40-07a305807b42.png)

It seems, that everyone in our LDAP system can log in. Sure, they have no permissions to see any mailbox. So it is not critical, but not as intended.

**Comment by freescout-helpdesk** [comment]
We'll check it.

**Comment by tiredofit** [comment]
Hows about this? `dc=de,dc=ct-srv(memberOf=cn=freescout,cn=groups,cn=accounts)`

**Comment by freescout-helpdesk** [comment]
What is your Bind DN?

LDAP module first tries to authenticate a user against "Bind DN", if not successful it tries all the "DNs and Filters". If your Bind DN is `dc=example,dc=org` for example and "DNs and Filters" is `cn=freescout,dc=example,dc=org` all users located below `dc=example,dc=org` will be able to authenticate. To avoid this you need to change your Bind DN to something like `cn=admins,dc=example,dc=org` and move the admin user there.

**Comment by freescout-helpdesk** [comment]
Closing for no activity.

**Comment by Brasil083** [comment]
Hi, the problem returns
if I add this line DC=example,DC=com(&(objectclass=top)(memberof=CN=example_group,OU=Example,DC=example,DC=com)) the filter matches the user to the group for import but grants access to all users. 
if I set something on the OU filter, with a group in the example you will not recognize anything for import.
Apparently it is mandatory to have users in the specified organizational unit

The goal is to restrict the import and allow access to a specific user in the security group
even if the base dn is the entire domain.
I think this was the purpose of the open and answered question, sorry if I made a mistake in commenting and didn't open a new question
